examples/cluster-claim.yaml

```diff
@@ -1,18 +1,22 @@
 apiVersion: aws.platformref.upbound.io/v1alpha1
 kind: Cluster
 metadata:
+  annotations:
+    render.crossplane.io/composition-path: apis/pat/composition.yaml
+    render.crossplane.io/function-path: examples/functions.yaml
   name: platform-ref-aws
   namespace: default
+  labels:
+    platform.upbound.io/deletion-ordering: enabled
 spec:
   compositeDeletePolicy: Foreground
   parameters:
     id: platform-ref-aws
     region: us-west-2
     version: "1.27"
-    iam:
+    #iam:
       # replace with your custom arn like:
-      # roleArn: arn:aws:iam::123456789:role/AWSReservedSSO_AdministratorAccess_d703c73ed340fde7
-      roleArn: ${data.aws_eks_iam_default_admin}
+      # principalArn: arn:aws:iam::123456789:role/AWSReservedSSO_AdministratorAccess_d703c73ed340fde7
```

Comment:
is this intentional?

Reply:
yes, this was failing for me after upgrade to most recent eks configuration

Reply:
Was renamed [here](https://github.com/upbound/configuration-aws-eks/commit/801fe5c989f569bf9a3993891bef7bdaa6b7615d#diff-e1925e528d29217f402dabddc19ed811c83cb387308b0086955e923c22a4bae7R39). After i took over the new name it was failing with something like invalid arn on aws-side so i decided to comment it out